### PR TITLE
changed signs in repository

### PR DESCRIPTION
**Pull request by anistal**
#### Description
Changed signs in RepositoryComponent in order to have a more reliable API used in CRUD operations.

### CODE REVIEW

File: src/main/scala/com/stratio/common/repository/zookeeper/ZookeeperRepositoryComponent.scala [hunk 1]
```diff
@@ -42,57 +45,50 @@ trait ZookeeperRepositoryComponent extends RepositoryComponent[String, Array[Byt
...
+        .forPath(id)
+      ).isDefined
+
+    def create(id: String, element: Array[Byte]): Array[Byte] = {
+      curatorClient
+        .create()
+        .creatingParentsIfNeeded()
+        .forPath(id, element)
+
+      get(id) match {
```
Comment (dvallejo) [inline src/main/scala/com/stratio/common/repository/zookeeper/ZookeeperRepositoryComponent.scala]: This pattern matching could be a getOrElse method

File: src/test/scala/com/stratio/common/repository/DummyRepositoryComponent.scala [hunk 2]
```diff
@@ -30,46 +28,37 @@ trait DummyRepositoryComponent extends RepositoryComponent[String, String] {
...
-        true
-      } else false
-    }
+    def exists(id: String): Boolean =
+      memoryMap.contains(id)
 
-    def update(id: String, element: String): Try[Boolean] = Try {
-      if(exists(id) == Success(true)) {
+    def create(id: String, element: String): String = {
+      if (exists(id) == false) {
```
Comment (dvallejo) [inline src/test/scala/com/stratio/common/repository/DummyRepositoryComponent.scala]: Boolean sentence could be simply 
```scala 
if (!exists(id))
```

File: src/test/scala/com/stratio/common/repository/DummyRepositoryComponent.scala [hunk 3]
```diff
@@ -30,46 +28,37 @@ trait DummyRepositoryComponent extends RepositoryComponent[String, String] {
...
     }
 
-    def delete(id: String): Try[Boolean] = Try {
-      if(exists(id) == Success(true)) {
-        memoryMap = memoryMap - id
-        true
-      } else false
-    }
+    def update(id: String, element: String): Unit =
+      if (exists(id) == true) memoryMap = memoryMap + (id -> element)
```
Comment (dvallejo) [inline src/test/scala/com/stratio/common/repository/DummyRepositoryComponent.scala]: Same as above

File: src/test/scala/com/stratio/common/repository/DummyRepositoryComponent.scala [hunk 4]
```diff
@@ -30,46 +28,37 @@ trait DummyRepositoryComponent extends RepositoryComponent[String, String] {
...
+
+    def delete(id: String): Unit =
+      if (exists(id) == true) memoryMap = memoryMap - id
```
Comment (dvallejo) [inline src/test/scala/com/stratio/common/repository/DummyRepositoryComponent.scala]: Same as above

**Comment by dvallejo** [comment]
After changes, LGTM. Good work!! :)

**Comment by gserranojc** [comment]
:+1:

**Pull request merged**